pantheon-decoupled.php

```diff
@@ -84,10 +124,42 @@ function pantheon_decoupled_options_page() {
                     </a>
                 </li>
             </ul>
-
         </div>
     <?php
 }
 
+/**
+ * HTML for list preview sites settings page.
+ *
+ * @return void
+ */
+function pantheon_decoupled_preview_list_html() {
+	// Check if the List_Table class is available.
+	if ( ! class_exists( 'WP_List_Table' ) ) {
+		require_once ABSPATH . 'wp-admin/includes/class-wp-list-table.php';
+	}
+	require_once plugin_dir_path( __FILE__ ) . 'class-list-table.php';
```

Comment:
These require statements will likely need to change based on feedback earlier in the PR.

Reply:
@backlineint Do you know the structure these URL's will need to take? Since we are now referencing the `class-wp-list-table` in the `decoupled-preview` plugin instead of `wp-admin` I know this will need to change but am having trouble building out the correct URL. I believe `ABSPATH` will also need to be replaced but I'm not sure by what.